src/main/res/values/strings.xml

```diff
@@ -285,14 +285,26 @@ limitations under the License.
     <string name="image_stop">Stop</string>
     <string name="image_track">Track</string>
     <!-- Import -->
+    <string name="import_unsupported_format">Unsupported file format</string>
+    <string name="import_parser_error">Parser error: %1$s</string>
+    <string name="import_unable_to_import_file">Unable to import file: %1$s</string>
+    <string name="import_error_info">%1$s: %2$s</string>
+    <string name="import_error_list_dialog_title">Files not imported</string>
+    <string name="import_file_imported">File %1$s imported</string>
     <string name="import_error">Imported %1$d of %2$s from %3$s</string>
     <string name="import_no_directory">%1$s does not exist</string>
     <string name="import_no_file">There were no files to import. Connect your device to your computer and place the files to import in %1$s.</string>
     <string name="import_no_file_title">Import files</string>
-    <string name="import_progress_message">Importing from %1$s&#8230;</string>
```

Comment:
IntelliJ's code analyze complains about this change.
The &#8230; is actually the correct thing for "..."